pkgs/by-name/gd/gdbuspp/package.nix

```
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchgit, meson, ninja, glib, cmake, pkg-config }:
+stdenv.mkDerivation rec {
+  pname = "gdbuspp";
+  version = "1";
+  src = fetchgit {
+    url = "https://codeberg.org/OpenVPN/gdbuspp.git";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-vw+37RbKRsB+DUyQU+ibwBHCj4jH/FaGl/bGSx7nrwY=";
+  };
+
+  postPatch = ''
+    patchShebangs --build ./scripts/get-git-ref
+  '';
+
+  nativeBuildInputs = [ meson ninja cmake pkg-config ];
```

```suggestion
  nativeBuildInputs = [ cmake pkg-config ];
```
Did you get mislead by the cmake errors? it should be enough to have at most 2 of those